libretro-common/vfs/vfs_implementation.c

```diff
@@ -491,18 +510,6 @@ libretro_vfs_implementation_file *retro_vfs_file_open_impl(
    stream->size = orbisLseek(stream->fd, 0, SEEK_END);
    orbisLseek(stream->fd, 0, SEEK_SET);
 #else
-#ifdef HAVE_CDROM
-   if (stream->scheme == VFS_SCHEME_CDROM)
-   {
-      retro_vfs_file_seek_cdrom(stream, 0, SEEK_SET);
-      retro_vfs_file_seek_cdrom(stream, 0, SEEK_END);
-
-      stream->size = retro_vfs_file_tell_impl(stream);
-
-      retro_vfs_file_seek_cdrom(stream, 0, SEEK_SET);
```

Comment:
`reftro_vfs_file_seek_internal` now has a switch case for `VFS_SCHEME_CDROM`, eliminating the need to have specific logic here.